bin/setup

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env sh
+
+# Exit if any subcommand fails
+set -e
```

Comment:
Can we `set -ex` and remove the echos below?

Reply:
Whaaat I never knew about `set -x`. That's cool.

Having tried it, it is slightly different than the echos:
- It shows all commands, even the `command -v mise` which is useful if you're debugging exactly what's happening, but is too verbose for navigating all the text scrolling down your terminal.
- It can't do the little bits of editorializing I put in the echos (pointing out that `mix ecto.setup` is in `(dev)`, and combining `local.hex and local.rebar`)

What do you think about adding it to `bin/test`, but keeping the echos in `bin/setup`?

Reply:
Points taken! That works for me 👍